app/assets/javascripts/alchemy/alchemy.node_tree.js

```diff
@@ -0,0 +1,93 @@
+Alchemy.NodeTree = {
+  onFinishDragging: function (evt) {
+    var url = '/api/nodes/' + evt.item.dataset.id + '/move.json'
```

Comment:
We need to pass in the URL from the erb template or add a new entry into the `app/views/alchemy/admin/partials/_routes.html.erb` file, because Alchemy might be mounted on a different path.